### PR TITLE
Halt Cron jobs between 01:00 and 03:00 on wednesdays

### PR DESCRIPTION
**Pull request by thinkwebengineer**
## Description

Added an if elif bash statement to check for date and time of the server and exit the cron job if it is between 01:00 and 03:00 server time on Wednesdays.

## Motivation and Context

A manual maintenance window is needed for the database between 01:00 and 03:00 on Wednesdays.  This requires cron jobs to be halted for clean handling. 

## Checklist:

- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docker/simplified_crontab [hunk 1]
```diff
@@ -78,7 +78,14 @@
 #   /var/log/simplified that's named after the script it was asked to run. Those
 #   messages will typically NOT appear in the container output or the cron.log file.
 #
-##############################################################################
+############################################################################### # A database maintenance window is needed between 01:00 and 03:00 on Wednesdays
+if [[ $(date +%u) -eq 3 && $(date +%H) -eq 1 ]]; then
+  echo "I don't run at this time"
+  exit 1
+elif [[ $(date +%u) -eq 3 && $(date +%H) -eq 1 ]]; then
```
Comment (keithbauer) [inline docker/simplified_crontab]: I think this line's hour (L#85) should be a 2.

**Comment by keithbauer** [comment]
I like this approach since it's tidy and I think we can confirm if this works locally by changing the hours to the current one then checking the docker logs to see if anything ran with `docker logs cm_local_scripts`.

File: docker/simplified_crontab (hunk 1 above)
Comment (thinkwebengineer) [reply docker/simplified_crontab]: devil is in the details lol. Thanks for the catch.

**Comment by thinkwebengineer** [comment]
This beautiful easy simple fix... breaks the docker cron build... closing PR to fix.

**Comment by thinkwebengineer** [comment]
Updated correct file with time check.  Files now build and the cron manager exits every time it is called for those 2 hours

**Review by keithbauer — APPROVED** [review]
This looks good to me now!

File: core/bin/run [hunk 2]
```diff
@@ -2,6 +2,14 @@
 
 #### Housekeeping functions and variables ####################################
 
+if [[ $(date +%u) -eq 3 && $(date +%H) -eq 1 ]]; then
+  echo "I don't run at this time"
+  exit 1
```
Comment (mwbenowitz) [inline core/bin/run]: should our exit code be `0` here? I don't think we consider this an exception, this is our intended functionality during this window.

File: core/bin/run [hunk 3]
```diff
@@ -2,6 +2,14 @@
 
 #### Housekeeping functions and variables ####################################
 
+if [[ $(date +%u) -eq 3 && $(date +%H) -eq 1 ]]; then
```
Comment (mwbenowitz) [inline core/bin/run:6]: Let's add a comment up top here explaining what this does and why for future reference!

**Review by mwbenowitz — COMMENTED** [review]
Functionality looks great! Two QoL questions/requests

File: core/bin/run (hunk 3 above)
Comment (thinkwebengineer) [reply core/bin/run:6]: Gotta love how fast the documentation guy forgets documentation :-/. Thanks!

**Review by mwbenowitz — APPROVED** [review]

**Pull request merged**